src/main/java/org/opendatakit/aggregate/client/FiltersDataPanel.java

```diff
@@ -91,12 +90,35 @@ public FiltersDataPanel(FilterSubTab parentPanel) {
     filtersTree = new Tree();
     panel.add(filtersTree);
 
+    String latestVersion = getLatestVersion();
+    boolean needsUpdate = !BuildConfig.VERSION.startsWith(latestVersion);

```

Comment:
Is this `startsWith` instead of `equals` so that someone running a local version with a commit hash isn't told to upgrade?

Reply:
Yes, that's right